wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,78 @@
+# How to install drivers and fix them 
+Hello, in this post you will know how to download or update your drivers for your computer.
+It's highly recommended to not use third-party software to update drivers.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly identify the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager (Windows) (right click into windows icon and press “Device Manager”). In the Device Manager, look for the device you require drivers for, and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you require drivers for.
+
+## Method 1: Check Windows Update
+You can usually find drivers and drivers update in Windows Update, instead of manually downloading the file.
+ 1. Open the Settings app using the Start menu.
+ 2. Go to Updates and security.
+ 3. Click on “Check updates”.
+ 4. Windows Update might say that you are up-to-date, even if optional updates are available. In this case, you will see an “Optional Updates” button. Clicking on it will show a list of updates (all of them are typically driver updates). Select all of them and click on “Download an installation”.
+
+## Method 2: Download using the manufacturer's website.
+It's highly recommended to download or update the drivers, through the manufacturer's website, as it's guaranteed that they will work correctly. This is especially a good idea for the GPU (Graphics) drivers.
+All you need to do is:
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{device manufacturer} drivers".
+ 2. Next, enter the ID of the device and follow the instructions on the page.
+ 3. When you selected the product, you have two options:
+	- Download the driver downloader (recommended)
+	- And download the driver on specific and run executable
+
+## Method 3: Download using the OEM's website.
+If you want to download the drivers for your GPU (Graphics Card), and the manufacturer's website doesn't provide the drivers, you can download them from the OEM's website. Only do this if you have to.
+Now follow the same instructions as Method 2, but with the OEM's website.
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{your brand device name} drivers".
```

Comment:
maybe change that search thing at the end to {the OEM's name} instead of {your brand device name}